### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Docs_Overview Left Navigation_AI4D: Name and role given for set up button is incorrect.

### ISSUE THREAD

**Issue by mspatil110** [microsoft/vscode-docs#3602]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#CELAP1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.3.1;#MAS4.1.2;#MAS4.2.1;#Win10;#Visual Studio Code;#CELAP1_code;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version  84.0.488.1 (Official build) dev (64-bit)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/
2)Visual Studio Code will get open.
3)Click on "docs" in the menu bar.
4)Go to "Overview Left Navigation".
5)Launch AI4D.
6)Verify the name and role type given for set up, get started buttons.

**Actual:**
Issue1: Name given for set up button is "Set up drop down ". Name text should not content any control type.

Issue2: Incorrect role type is given for set up button. Role type is given as a link but it is a button.

**Same issue has been observed for all the buttons present under the heading overview on same page.**

**Expected:**
Issue1: Name text of set up button should not content any role type. Name should be "Set up" button.

Isssue2: Role type button should be present for set up button.

**User Impact:**
Screen reader only users will face difficulty to understand the buttons properly.

**Attachment link for Reference:**
![3602_CELA P1_A11y_code visualstudio com_VisualStudioCode_Docs_Overview Left Navigation_AI4D](https://user-images.githubusercontent.com/51433293/80100298-659ea480-858d-11ea-8a96-b0c5f7690c76.png)

**Comment by gregvanl** [comment]
Conflicts with fix for eariler issue https://github.com/microsoft/vscode-docs/issues/2810

**Comment by khansahab15** [comment]
GitHubTags:#P1_A11y_April2020

**Comment by ManishaC1** [comment]
Verified the issue in below environment.
URL: https://code.visualstudio.com/
Issue if fixed.
Github tags: #Closed
![Screenshot (427)](https://user-images.githubusercontent.com/105198219/216914520-dd61a0fe-4b24-4053-92df-4782f7694619.png)

**Comment by fsteffi** [comment]
githubtags:#P1_A11y_April2020;

**Comment by mstechie** [comment]
GitHubTags:#P1_A11y_code.visualstudio.com_Apr2020
